### PR TITLE
Issue92 cleanup example

### PR DESCRIPTION
**Pull request by yescorp**

### CODE REVIEW

File: example/lib/03.control/code_editor.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _setLanguage(String value) {
+    setState(() {
+      _language = value;
+      _codeController = codeControllers[_language]!;
```
Comment (yescorp) [inline example/lib/03.control/code_editor.dart:60]: It doesn't reflect the UI if I only change language property in the codeController, like:
`_codeController.language = builtinLanguages[value];`

Still looking for a better way to do it.

File: example/lib/03.change_language_theme/constants/constants.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import 'package:highlight/languages/dart.dart';
+import 'package:highlight/languages/go.dart';
+import 'package:highlight/languages/java.dart';
+import 'package:highlight/languages/python.dart';
+import 'package:highlight/languages/scala.dart';
+
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/constants/constants.dart:6]: Move `../`

File: example/lib/03.change_language_theme/constants/constants.dart [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+final builtinLanguages = {
+  'go': go,
+  'java': java,
+  'python': python,
+  'scala': scala,
+  'dart': dart,
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/constants/constants.dart:12]: Sort.

File: example/lib/03.change_language_theme/home_screen.dart [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+import '../common/snippets.dart';
+import '../common/themes.dart';
+import 'constants/constants.dart';
+import 'widgets/dropdown_selector.dart';
+
+class HomeScreen extends StatefulWidget {
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:13]:
```dart
const _defaultLanguage = 'java';
const _defaultTheme = 'monokai-sublime';
```

File: example/lib/03.change_language_theme/constants/constants.dart [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+const languageList = <String>[
+  'java',
+  'go',
+  'python',
+  'scala',
+  'dart',
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/constants/constants.dart:20]: Sort.

File: example/lib/03.change_language_theme/constants/constants.dart [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+];
+
+const themeList = <String>[
+  'monokai-sublime',
+  'a11y-dark',
+  'an-old-hope',
+  'vs2015',
+  'vs',
+  'atom-one-dark',
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/constants/constants.dart:29]: Sort.

File: example/lib/03.change_language_theme/home_screen.dart [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Code Editor by Akvelon'),
+        actions: [
+          // Language Selector
+          DropdownSelector(
+              onChanged: _setLanguage,
+              icon: Icons.code,
+              value: _language,
+              values: languageList),
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:35]: Trailing comma.

File: example/lib/03.change_language_theme/home_screen.dart [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+    text: javaFactorialSnippet,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Code Editor by Akvelon'),
+        actions: [
+          // Language Selector
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:30]: Delete.

File: example/lib/03.change_language_theme/home_screen.dart [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+          DropdownSelector(
+              onChanged: _setLanguage,
+              icon: Icons.code,
+              value: _language,
+              values: languageList),
+          const SizedBox(
+            width: 20,
+          ),
+          // Theme selector
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:39]: Delete.

File: example/lib/03.change_language_theme/home_screen.dart [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+        actions: [
+          // Language Selector
+          DropdownSelector(
+              onChanged: _setLanguage,
+              icon: Icons.code,
+              value: _language,
+              values: languageList),
+          const SizedBox(
+            width: 20,
+          ),
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:38]: To a single line.

File: example/lib/03.change_language_theme/widgets/dropdown_selector.dart [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+          value: value,
+          child: Text(value, style: const TextStyle(color: Colors.white)),
+        );
+      }).toList(growable: false),
+      icon: Icon(icon, color: Colors.white),
+      onChanged: (value) { if (value != null) onChanged(value); },
+      dropdownColor: Colors.black87,
+    );
+  }
+}
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/widgets/dropdown_selector.dart:31]: https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e

**Review by alexeyinkin — CHANGES_REQUESTED** [review]

File: example/lib/03.change_language_theme/constants.dart [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+  'scala',
+];
+
+const themeList = <String>[
+  'a11y-dark',
+  'an-old-hope',
+  'atom-one-dark',
+  'monokai-sublime',
+  'vs2015',
+  'vs',
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/constants.dart:29]: Swap.

**Review by alexeyinkin — CHANGES_REQUESTED** [review]

File: example/lib/03.change_language_theme/home_screen.dart [hunk 13]
```diff
@@ -69,18 +70,21 @@ class _HomeScreenState extends State<HomeScreen> {
   void dispose() {
     super.dispose();
```
Comment (alexeyinkin) [inline example/lib/03.change_language_theme/home_screen.dart:71]: `super.dispose();` should be called last.

**Review by alexeyinkin — CHANGES_REQUESTED** [review]

**Review by alexeyinkin — APPROVED** [review]

**Pull request merged**